### PR TITLE
Update tile_matrix_sets.md

### PR DESCRIPTION
**Pull request by RichardScottOZ**
WIP Edits I will add more to this one, so leave and I will ping when done
Note that the notebook link at the end 404s similarly

### CODE REVIEW

**Comment by RichardScottOZ** [comment]
@vincentsarago this is good to go now (with the couple before as well)

File: docs/concepts/output_format.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
 `TiTiler` supports the common output format for map tiles: JPEG, PNG and WEBP.
 
-While some format (e.g PNG) are able to encode Uint16 or Float datatypes, most web browser only supports 8 bytes data (meaning that it has to be between 0 and 255).
+While some formats (e.g PNG) are able to encode Uint16 or Float datatypes, most web browsers only supports 8 byte data (meaning that it has to be between 0 and 255).
```
Comment (kylebarron) [inline docs/concepts/output_format.md:4]:
```suggestion
While some formats (e.g PNG) are able to encode Uint16 or Float datatypes, most web browsers only supports 8 bit data (meaning that it has to be between 0 and 255).
```

File: docs/concepts/output_format.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Default output types/extensions are:
 
 ## NumpyTile
 
-While `.tif` could be interesting, decoding the `GeoTIFF` format in requires non-native/default libraries. Recently, in colaboration with Planet, we started exploring the use of Python [`Numpy Format`](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html#format-version-1-0) to encode the data array.
+While `.tif` could be interesting, decoding the `GeoTIFF` format requires non-native/default libraries. Recently, in colaboration with Planet, we started exploring the use of Python [`Numpy Format`](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html#format-version-1-0) to encode the data array.
```
Comment (kylebarron) [inline docs/concepts/output_format.md:21]:
```suggestion
While `.tif` could be interesting, decoding the `GeoTIFF` format requires non-native/default libraries. Recently, in collaboration with Planet, we started exploring the use of a [`Numpy-native format`](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html#format-version-1-0) to encode the data array.
```

File: docs/concepts/tiler_factories.md [hunk 3]
```diff
@@ -32,6 +32,6 @@ Tiler factories (`titiler.endpoints.factory.TilerFactory|MosaicTilerFactory`) ar
 | `GET`  | `/point/{lon},{lat}`                                            | JSON      | return pixel value from a MosaicJSON dataset
 
 
-**Factories** are built around [`rio_tiler.io.BaseReader`](https://cogeotiff.github.io/rio-tiler/advanced/custom_readers/), which define basics method to access datasets (e.g COG or STAC). The default reader is `COGReader` for `TilerFactory` and `MosaicBackend` for `MosaicTilerFactory`.
+**Factories** are built around [`rio_tiler.io.BaseReader`](https://cogeotiff.github.io/rio-tiler/advanced/custom_readers/), which define a basic method to access datasets (e.g COG or STAC). The default reader is `COGReader` for `TilerFactory` and `MosaicBackend` for `MosaicTilerFactory`.
```
Comment (kylebarron) [inline docs/concepts/tiler_factories.md:35]:
```suggestion
**Factories** are built around [`rio_tiler.io.BaseReader`](https://cogeotiff.github.io/rio-tiler/advanced/custom_readers/), which defines basic methods to access datasets (e.g COG or STAC). The default reader is `COGReader` for `TilerFactory` and `MosaicBackend` for `MosaicTilerFactory`.
```

**Review by kylebarron — APPROVED** [review]
🙏 So many fixes!

**Pull request merged**